blueocean-dashboard/src/main/js/components/PipelinePage.jsx

```diff
@@ -84,9 +85,9 @@ export class PipelinePage extends Component {
                     </Title>
                     }
                     <PageTabs base={baseUrl}>
-                        <TabLink to="/activity">Activity</TabLink>
-                        <TabLink to="/branches">Branches</TabLink>
-                        <TabLink to="/pr">Pull Requests</TabLink>
+                        <TabLink to="/activity">t('bo.dashboard.activity')</TabLink>
+                        <TabLink to="/branches">t('bo.dashboard.branches')</TabLink>
+                        <TabLink to="/pr">t('bo.dashboard.pr')</TabLink>
```

Comment:
NIT .... maybe we can drop the `bo.` prefix from the keys .... not really needed I think, no?